packages/ui/src/components/va-data-table/hooks/useTableScroll.ts

```diff
@@ -0,0 +1,35 @@
+interface useTableScrollProps {
```

Comment:
Maybe will be better to create exported object `useTableScrollProps`, then `props: ExtractPropTypes<typeof useTableScrollProps>` and  import-destruct it in `VaDataTable` props.